sentry.client.config.js

```diff
@@ -7,4 +7,5 @@ Sentry.init({
   environment: process.env.NEXT_PUBLIC_SENTRY_ENVIRONMENT ?? "unset",
   dsn,
   tracesSampleRate: 1,
+  tunnel: "/api/sentry-tunnel",
```

Comment:
@PiRomant can ad-blockers ban this route because it mentions Sentry? We can go for `/api/tunnel` if so.